docker_mcp/core/safety.py

```
@@ -0,0 +1,244 @@
+            file_path: Path to be deleted
+            operation: Type of operation (rm, rm -f, rm -rf, etc.)
+            reason: Reason for deletion
+        """
+        manifest_entry = {
+            "path": file_path,
+            "operation": operation,
+            "reason": reason,
+            "timestamp": asyncio.get_event_loop().time(),
+            "validated": False
+        }
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)